apps/web/src/pages/ApplicantDashboardPage/components/MetadataDate.tsx

```diff
@@ -0,0 +1,152 @@
+  if (isReceivedStatus || isUnderReviewStatus || isUnderAssessmentStatus) {
+    return (
+      <span>
+        {intl.formatMessage({
+          defaultMessage: "Submitted",
+          id: "B1sXVl",
+          description: "Label for submitted metadata",
+        })}{" "}
+        {intl.formatMessage(commonMessages.dividingColon)}{" "}
+        {submittedAt
+          ? formatDate({
+              date: parseDateTimeUtc(submittedAt),
+              formatString: "PPP",
+              intl,
+              timeZone: "Canada/Pacific",
+            })
+          : nullMessage}
+      </span>
+    );
+  }
+
+  const isSuccessfulStatus = status === ApplicationStatus.SUCCESSFUL;
+  const isUnsuccessfulStatus = status === ApplicationStatus.UNSUCCESSFUL;
+  if (isSuccessfulStatus || isUnsuccessfulStatus) {
+    return (
+      <span>
+        {intl.formatMessage({
+          defaultMessage: "Assessed",
+          id: "GYcxDu",
+          description: "Label for assessed metadata",
+        })}{" "}
+        {intl.formatMessage(commonMessages.dividingColon)}{" "}
+        {finalDecisionAt
+          ? formatDate({
+              date: parseDateTimeUtc(finalDecisionAt),
+              formatString: "PPP",
+              intl,
+              timeZone: "Canada/Pacific",
+            })
+          : nullMessage}
+      </span>
+    );
+  }
+
+  return null;
+};
+
+export const RecruitmentDate = ({
+  finalDecisionAt,
+  removedAt,
+  status,
+}: {
+  finalDecisionAt?: string | null;
+  removedAt?: string | null;
+  status: ApplicationStatus;
+}) => {
+  const intl = useIntl();
+  const nullMessage = intl.formatMessage(commonMessages.notFound);
+
+  const isOpenToJobsStatus = status === ApplicationStatus.OPEN_TO_JOBS;
+  const isNotInterestedStatus = status === ApplicationStatus.NOT_INTERESTED;
+  const isHiredStatus = status === ApplicationStatus.HIRED;
+  const isExpiredStatus = status === ApplicationStatus.EXPIRED;
+  if (
+    isOpenToJobsStatus ||
+    isNotInterestedStatus ||
+    isHiredStatus ||
+    isExpiredStatus
+  ) {
+    return (
+      <span>
+        {intl.formatMessage(commonMessages.awarded)}{" "}
+        {intl.formatMessage(commonMessages.dividingColon)}{" "}
+        {finalDecisionAt
+          ? formatDate({
+              date: parseDateTimeUtc(finalDecisionAt),
+              formatString: "PPP",
+              intl,
+              timeZone: "Canada/Pacific",
+            })
+          : nullMessage}
+      </span>
+    );
+  }
+
+  const isUnsuccessfulStatus = status === ApplicationStatus.UNSUCCESSFUL;
+  if (isUnsuccessfulStatus) {
+    return (
+      <span>
+        {intl.formatMessage(commonMessages.removed)}{" "}
+        {intl.formatMessage(commonMessages.dividingColon)}{" "}
+        {removedAt
+          ? formatDate({
+              date: parseDateTimeUtc(removedAt),
+              formatString: "PPP",
+              intl,
+              timeZone: "Canada/Pacific",
```

Comment:
Is this supposed to be pacific time? I fee like it makes more sense to use the users local timezone but I may be wrong

Reply:
Yup fixed in [33de8ab](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/33de8ab0c0cddbced251c5a5ae1cd8122902367e)